src/data-workspace/data-details-sidebar/basic-information.js

```diff
@@ -2,68 +2,82 @@ import i18n from '@dhis2/d2-i18n'
 import { Tooltip, IconFlag16, colors } from '@dhis2/ui'
 import moment from 'moment'
 import React from 'react'
+import { useServerDateAtClientTimezone } from '../../shared/index.js'
 import FollowUpButton from './basic-information-follow-up-button.js'
 import styles from './basic-information.module.css'
 import ItemPropType from './item-prop-type.js'
 
-const BasicInformation = ({ item }) => (
-    <div className={styles.unit}>
-        <h1 className={styles.title}>
-            {item.displayFormName}
-            {item.categoryOptionComboName &&
-                ` | ${item.categoryOptionComboName}`}
-        </h1>
+const BasicInformation = ({ item }) => {
+    // This might pass "undefined" to moment and subsequently a wrong
+    // "timeAgo", but in that case we won't render anything anyway, so there's
+    // nothing to worry about in case there is no "item"
+    const lastUpdatedUtcDate = new Date(item.lastUpdated)
```

Comment:
should the variable name be updated to not reference UTC?

Reply:
addressed